src/containers/Dashboard/Forms/QuizForm.js

```diff
@@ -0,0 +1,142 @@
+    this.state = {
+      name: "",
+      courseId: ""
+    };
+    this.handleUserInput = this.handleUserInput.bind(this);
+    this.notificationDOMRef = React.createRef();
+    this.submitForm = this.submitForm.bind(this);
+    this.formRef = null;
+  }
+
+  handleUserInput(e) {
+    const name = e.target.name;
+    const value = e.target.value;
+    this.setState({ [name]: value });
+  }
+  submitForm(e) {
+    e.preventDefault();
+    if (this.validator.allValid()) {
+      var { name, courseId } = this.state;
+      const data = {
+        name,
+        courseId
+      };
+      API.quizData(data, result => {
+        if (result.status === "201") {
+          // Form reset
+          this.formRef.reset();
+          addNotification(
+            this.notificationDOMRef,
+            "success",
+            "success",
+            result.message
+          );
+        } else if (
+          result.status === "404" ||
+          result.status === "400" ||
+          result.status === "500"
+        ) {
+          addNotification(
+            this.notificationDOMRef,
+            "Error",
+            "danger",
+            result.status
+          );
+        }
+      }).catch = error => {
+        addNotification(this.notificationDOMRef, "Error", "warning", error);
+      };
+    } else {
+      this.validator.showMessages();
+      // rerender to show messages for the first time
+      this.forceUpdate();
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <div>
+          <ReactNotification ref={this.notificationDOMRef} />
+        </div>
+        <div className="d-flex justify-content-center container">
+          <div className="row">
+            <div className="form-inner-container">
+              <h1 className="heading" align="center">
+                Quiz Creation Form
+              </h1>
+              <form
+                onSubmit={this.submitForm}
+                ref={ref => (this.formRef = ref)}
+              >
+                <div>
+                  <label className="labels">Name</label>
+                  <br />
+                  <input
+                    className="form-field"
+                    name="name"
+                    type="text"
+                    onChange={this.handleUserInput}
+                  />
+                  <div className="form-error-msg">
+                    {this.validator.message(
+                      "name",
+                      this.state.name,
+                      "min:3|max:25|required"
+                    )}
+                  </div>
+                </div>
+                <div>
+                  <label className="labels">Course</label>
+
+                  <select
+                    className="custom-select"
+                    name="courseId"
+                    type="text"
+                    onChange={this.handleUserInput}
+                  >
+                    <option defaultValue>Select Course</option>
```

Comment:
@Humnaa 
Remove this hard coded options in `course` selection. You now must be getting this data from `api` as you can add `course` now.

Reply:
I'm getting dynamic data from API `/course/new` and showing in select fields.